app/src/components/Provenance.tsx

```diff
@@ -135,7 +158,9 @@ class Provenance extends React.Component<ProvenanceProps, ProvenanceState> {
             <Modal.Title>Provenance</Modal.Title>
           </Modal.Header>
           <Modal.Body>
-            <Timeline records={this.state.saleProvenance}/>
+            <Timeline accounts={this.props.accounts}
+              contracts={this.props.contracts}
+              records={this.state.saleProvenance}/>
```

Comment:
Probably should provide this stuff with Contexts to prevent having to thread all these through, but it's not wrong